core/src/View/Kokkos_ViewCtor.hpp

```diff
@@ -27,6 +27,22 @@ struct SequentialHostInit_t {};
 struct WithoutInitializing_t {};
 struct AllowPadding_t {};
 
+// FIXME: AccessorArg_t needs to be templated
+// This really needs to be templated on the argument type
+// But that adds some more complication, preventing simply
+// copying the code from the above properties.
+// For Sacado we only need a size_t for fad_size
+// We can template this later in a separate change.
```

Comment:
Can you briefly explain why we want an argument to the accessor constructor rather than store the accessor directly?